examples/character-io/character_io.bal

```diff
@@ -1,87 +1,48 @@
 import ballerina/io;
+import ballerina/log;
 
-// This function reads characters from 'charChannel',
-//which is an instance of ReadableCharacterChannel.
-function readCharacters(io:ReadableCharacterChannel charChannel,
-                        int numberOfChars) returns string {
-    //This is how the characters are read.
-    match charChannel.read(numberOfChars) {
-        string characters => {
-            return characters;
-        }
-        error err => {
-            throw err;
-        }
-    }
+// This function reads content from a file,
+// appends the additional string, and writes the content.
+function process(io:ReadableCharacterChannel sc,
+                 io:WritableCharacterChannel dc) returns error? {
+    // Here is the string that is appended in-between.
```

Comment:
We don't need to say "Here is". 

Can we expand on this comment slightly? Appended in-between what?